docs/concepts/packet.md

```diff
@@ -28,3 +28,8 @@ spec:
     - ipBlock:
         cidr: 0.0.0.0/0
 ```
+
+## Flatcar Linux Customization
+
+Flatcar Container Linux deployments on Packet can be customized with Container Linux Configs.
+For documentation, see the ["Customization" section of the Flatcar Container Linux document](/docs/concepts/flatcar-container-linux.md#Customization).
```

Comment:
I'd rather say "for more information" or "for more info". "For documentation" sounds strange to me.